sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -192,6 +194,9 @@ def _next_uid(self):
 
   def run_pipeline(self, pipeline):
     MetricsEnvironment.set_metrics_supported(False)
+    # This is sometimes needed if type checking is disabled.
```

Comment:
<!--new_thread; commit:b52c38538e216b86d905166341e4a98bf1ef72b9; resolved:0-->
Can you clarify in more detail what this does?